src/files.rs

```diff
@@ -42,6 +42,10 @@ pub fn clear_release(version: &String) -> Result<(), Error> {
         println!("  {} old release v{}", "Cleaning".yellow().bold(), version);
         fs::remove_dir_all(format!("releases/{}", version))?;
     }
+    if Path::new("releases/keys").exists() {
+        println!("  {} old keys", "Cleaning".yellow().bold());
+        fs::remove_dir_all("releases/keys")?;
```

Comment:
This would delete all of the keys ever - the private key if it exists, and all keys for versions other than the one being cleared. That would be surprising behaviour to me. I'd expect it only to remove keys associated with the release I'm trying to release.

Reply:
I tend to agree, but I was lazy, need to find the correctly named one here too. ^^

Reply:
```rust
Path::new(&format!("releases/keys/{}", p.get_keyname()))?;
```
I guess.

Reply:
But won't it just be overwritten anyway?

Reply:
Apparently not.